contrib/rclone/template.py

```diff
@@ -0,0 +1,51 @@
+pkgname = "rclone"
+pkgver = "1.64.2"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Rsync for cloud storage"
+maintainer = "flukey <flukey@vapourmail.eu>"
+license = "MIT"
+url = "https://rclone.org"
+source = f"https://github.com/rclone/rclone/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "85feffc2d60554bcc3c59140750dc4ccf008e109b52c451956a1f52387af1bd6"
+# tests require network
+options = ["!check"]
```

Comment:
since you're executing built artifacts, it should also be !cross

Reply:
alternatively, you can do something like @wezm did in https://github.com/chimera-linux/cports/pull/992